macros/edr/dbt_artifacts/upload_dbt_invocation.sql

```diff
@@ -25,6 +26,11 @@
       'selected': elementary.get_invocation_select_filter(),
       'yaml_selector': elementary.get_invocation_yaml_selector(),
       'project_name': elementary.get_project_name(),
+      'env': elementary.get_env(),
+      'run_reason_category': elementary.get_run_reason_category(),
+      'run_reason': elementary.get_run_reason(),
+      'pull_request_id': elementary.get_pr_id(),
+      'git_sha': elementary.get_git_sha(),
```

Comment:
Anything else we should add?
Here's [GitHub's envs](https://docs.github.com/en/actions/learn-github-actions/environment-variables).